modules/virt-using-skip-node.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assembly:
+//
+// * virt/node_maintenance/virt-using-skip-node.adoc
+//
+
+[id="virt-using-skip-node_{context}"]
+
+.Procedure
```

Comment:
You can also add a `.Prerequisites` section to the module, with this as the prereq (commonly used throughout the docs):
```
* You have installed the OpenShift CLI (`oc`).
```